README.md

```diff
@@ -15,7 +15,38 @@ The crate provides an API for working with containers in a test environment.
 
 1. Depend on `testcontainers`
 2. Implement `testcontainers::core::Image` for necessary docker-images
-3. Run it with any available client `testcontainers::clients::*`
+3. Run it with any available runner `testcontainers::runners::*` (use `blocking` feature for synchronous API)
+
+#### Example:
+
+- Blocking API (under `blocking` feature)
+
+```rust
+use testcontainers::{core::WaitFor, runners::SyncRunner, GenericImage};
+
+#[test]
+fn test_redis() {
+    let container = GenericImage::new("redis", "latest")
```

Comment:
I would not use `latest` here, since I personally would not recommend using `latest` in any test either due to (at least best effort) reproducibility.